### PR TITLE
OSDOCS-7781: Migrate Extend DNS for ROSA custom domain

### PR DESCRIPTION
**Pull request by michaelryanmcneill**
Version(s):
4.13+

Issue:
https://issues.redhat.com/browse/OSDOCS-7781

Link to docs preview:
https://65771--docspreview.netlify.app/openshift-rosa/latest/cloud_experts_tutorials/cloud-experts-external-dns

QE review:
- [ ] QE has approved this change.

Additional information:
Supersedes #64959

### CODE REVIEW

File: cloud_experts_tutorials/cloud-experts-external-dns.adoc [hunk 1]
```diff
@@ -0,0 +1,339 @@
...
+//weight: 1
+//tags: ["AWS", "ROSA"]
+//authors:
+//  - Chris Kang
+//  - Dustin Scott
+//---
+
+Configuring the xref:../applications/deployments/osd-config-custom-domains-applications.adoc[Custom Domain Operator] requires a wildcard CNAME DNS record in your Amazon Route 53 hosted zone. If you do not want to use a wildcard record, you can use the `External DNS` Operator to create individual entries for routes.
+
+This tutorial will guide you through deploying and configuring the External DNS Operator with a custom domain in {product-title} (ROSA).
```
Comment (bmcelvee) [inline cloud_experts_tutorials/cloud-experts-external-dns.adoc:23]: Suggested edit: 
This tutorial guides you...

File: cloud_experts_tutorials/cloud-experts-external-dns.adoc [hunk 2]
```diff
@@ -0,0 +1,339 @@
...
+
+. Apply the CR:
++
+.Example
+[source,terminal]
+----
+$ oc apply -f external-dns-custom-domain.yaml
+----
+
+. Verify that your custom domain ingress controller has been deployed and is `Ready`:
```
Comment (bmcelvee) [inline cloud_experts_tutorials/cloud-experts-external-dns.adoc:110]: s/ingress controller/Ingress Controller

**Comment by bmcelvee** [comment]
Just a couple of suggestions. Overall LGTM!

**Comment by michaelryanmcneill** [comment]
Changes pushed! Ready for merge (once the build completes) @bmcelvee. Thank you!

**Pull request merged**